torch/csrc/jit/frontend/ir_emitter.cpp

```diff
@@ -2751,9 +2751,13 @@ struct to_ir {
     // through RPC in TorchScript,
     // Ideally, function value in JIT IR is first-class citizen and
     // The RPC C++ entry API can take c10::Function directly.
-    if (apply.inputs().size() < 2 || apply.inputs().size() > 4) {
+    auto rpcMinInputs = 2;
+    auto rpcMaxInputs = 5;
```

Comment:
Hmm not sure how to fix this clang-tidy error, or why it's erroring for 5 and not 2...